ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -134,7 +169,7 @@ public void SetPointerEvents(BorderedCanvas view, string pointerEventsValue)
         public void SetBorderRadius(BorderedCanvas view, int index, double radius)
         {
             var border = GetOrCreateBorder(view);
-            var cornerRadius = border.CornerRadius == null ? new CornerRadius() : border.CornerRadius;
```

Comment:
> == null ? new CornerRadius() : border.CornerRadius; [](start = 50, length = 52)

What's the reasoning for this change?

Reply:
CornerRadius is a struct, so "== null" is false in any case, so the code I removed is dead code.